kateryna-tolochko/README.md

```diff
@@ -0,0 +1,91 @@
+
+   3a. If you choose to sign up through your social media accounts, you will be redirected to a page where you will need to give AliExpress permission to access your account information. Follow the on-screen instructions to complete the process.
+
+   3b. If you choose to sign up using your email, fill out the form. Input your valid email, password (6-20 symbols) and click **Create Account**. Check your email box and find the letter from AliExpress. Click the verification link provided in the email to verify your account. If you did not receive email with account verification link, please check your spam folder.
+
+4. Once your account is successfully verified, you will be directed to the AliExpress website. You will be asked to provide additional information, such as your name and shipping address. Fill in the required fields accurately.
+
+## Step 2. Browse products
+
+Once you have created an account and logged in, you can start browsing for products. To search for a specific item, use the search bar located at the top of the page. Enter relevant keywords or product name and click the search icon. Alternatively, you can explore the various categories and subcategories available on the homepage to discover products of interest. Hover over a category to view subcategories and click the desired category to view products within it.
+
+![Categories and subcategories](https://i.ibb.co/XY0ZBkz/screen2.png)
+
+## Step 3. Refine your search
+
+After performing a search or selecting a category, you will see the search results page. To refine your search, use the available filters located at the top and left of the page. You can filter products by price range, rating, brand, shipping country, and more. Select the desired filters to narrow down your options and find the products that best suit your needs.
+
+![Filters](https://i.ibb.co/C2BHM3P/screen5.png)
+
+## Step 4. Select a product
+
+1. When you find a product that interests you, click it to view the product details page.
+
+2. Read the product description, specification, and reviews from previous customers.
+
+3. If the product has different variations (e.g., size, color), make sure to select the desired options before proceeding.
+
+## Step 5. Place an order
+
+1. On the product details page, click **Buy Now**.
+
+2. Provide shipping address.
+
+3. Select a method for payment. Enter the payment information, card info. Click **Confirm**.
+
+4. Click **Place Order**.
+
+## Step 6. Track your order
+
+After your payment is confirmed, the seller will prepare your order for shipment. Once the seller ships your order, you will be able to track the progress of your order.
+
+1. Hover over your account on the top right and go to **My Orders**.
+
+   !['Account' menu](https://i.ibb.co/Yj7Tbj2/screen3.png)
+
+2. Locate the item that you want to track and then click **Track order**.
+
+3. You will now be able to see the current status of your order and the AliExpress tracking number.
```

Comment:
Add the screenshot for an example if possible. Text with screenshots are better that the text itself.